vm/src/builtins/pytype.rs

```diff
@@ -337,7 +337,7 @@ impl PyType {
     }
 
     #[pyproperty(magic)]
-    fn module(&self, vm: &VirtualMachine) -> PyObjectRef {
+    pub fn module(&self, vm: &VirtualMachine) -> PyObjectRef {
```

Comment:
I should've made this clearer in the last comment, this should also be `pub(crate)`.